content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -774,6 +774,12 @@ Nodes as `NotReady` and `NoSchedule`. That tainting avoids a race condition that
 to be scheduled on new Nodes before their taints were updated to accurately reflect their reported
 conditions.
 
+### ValidatingAdmissionPolicy {#validatingadmissionpolicy}
+
+This admission controller implements the CEL validation for incoming matched requests. 
```

Comment:
(nit)
Please hyperlink to `/docs/reference/access-authn-authz/validating-admission-policy/` here.